code/api/src/main/scala/com/avast/datadog4s/api/event/Eventer.scala

```diff
@@ -0,0 +1,10 @@
+package com.avast.datadog4s.api.event
+
+import com.avast.datadog4s.api.Tag
+
+trait Eventer[F[_]] {
```

Comment:
i think we could simply incomporate this method into the `EventFactory`, similar to how we do this in `MetricsFactory` to simplify things a little